cdap-ui/app/cdap/components/PipelineDetails/PipelineDetailsTopPanel/PipelineDetailsDetailsActions/PipelineDetailsDetailsButton.js

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright © 2018 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+*/
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import IconSVG from 'components/IconSVG';
+import AbsLinkTo from 'components/AbsLinkTo';
+import {getCurrentNamespace} from 'services/NamespaceStore';
+
+export default function PipelineDetailsDetailsButton({pipelineName}) {
+  let context = {
+    namespaceId: getCurrentNamespace(),
+    appId: pipelineName
+  };
+
+  return (
+    <div className="pipeline-action-container pipeline-details-container">
+      <AbsLinkTo context={context}>
+        <div className="btn pipeline-action-btn pipeline-details-btn">
+          <div className="btn-container">
+            <IconSVG name="icon-info-circle"/>
```

Comment:
might not be in this PR but we might want to think about having a title prop for `IconSVG` . Right now when I hover over the icon-info-circle icon it shows `info-circle` as tooltip. Would be nice if we show some text related to showing details of the pipeline.

Reply:
@ajainarayanan A few thoughts regarding making the modeless dyanmic. The current design of the pipeline studio and pipeline details is not optimized for VM. Unless we have a conversation on how to make those pages responsive, I don’t see the point why we should invest time in making the modeless dynamic. I'll be happy to have that conversation and prioritize the design after 5.0